apis/apps/v1alpha1/componentdefinition_types.go

```diff
@@ -841,6 +854,13 @@ type Action struct {
 	// +optional
 	Exec *ExecAction `json:"exec,omitempty"`
 
+	// Specifies the GRPC request to perform.
+	//
+	// This field cannot be updated.
+	//
+	// +optional
+	GRPC *GRPCAction `json:"grpc,omitempty"`
```

Comment:
As a provider, how do I know what services need to be implemented?

Reply:
en, this is the problem, and other handlers have. To ensure clarity, it is essential to explicitly address this concern at the action level in the documentation.